analysis/analysis.py

```diff
@@ -51,6 +60,9 @@ def Energy_cells_distance(G, dCells,  Egg=1, Egr=1.5, Err=1):
   return E
 
 def mean_lengths(G):
+    """
+    From a networkx graph, return the average distance between cell core, depending on the color of each cells
+    """
   total_length = 0
```

Comment:
Make the inputs and outputs even clearer :)

```suggestion
  """
  INPUT:
   - G: networkx graph object
   
   RETURNS:
   - G: networkx graph with modified node attributes
  """
```